conf/config.js

```diff
@@ -81,6 +81,22 @@ const SELL_LIQUIDITY_BOTS = [{
   checkTimeInMilliseconds: 60 * 1000 // 60s
 }]
 
+const DEPOSIT_BOT = {
+  name: 'Deposit bot',
+  notifications: [{
+    type: 'slack',
+    channel: '' // If none provided uses SLACK_CHANNEL_BOT_TRANSACTIONS
+  }],
+  inactiveTimeLapse: [{
```

Comment:
Not strong opinion, but I like more inactivityWindows

Reply:
I'll go with inactivityPeriods

Reply:
ok!